cypress/e2e/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -215,10 +205,10 @@ function testTool(registry: string, repo: string, name: string) {
       cy.visit('/my-tools');
       goToVersionsTab();
       toggleHiddenToolVersion();
-      cy.get('[data-cy=hiddenCheck]').should('have.length', 1);
+      cy.get('.mat-column-hidden').contains('mat-icon', 'check').should('have.length', 1);
```

Comment:
Changed because the data-cy name changed so that the tool and workflow tables have the same data-cy name. This is only temporary and will be changed back to using data-cy when this PR is merged to develop.